.buildkite/hooks/pre-exit

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -euo pipefail
+[[ -n "${DEBUG:-}" ]] && set -x
+
+# The premerge step runs on linux agents!
+if [[ ${BUILDKITE_AGENT_META_DATA_OS} == "windows" ]]; then
```

Comment:
the comment says it runs on linux againets, but here we are checking for windows?

Reply:
Just copying across @jamiebrynes7 's PR: https://github.com/spatialos/gdk-for-unity/pull/744

Reply:
We only want to run the purge script on Windows - surprisingly the `premerge` step runs on a Linux box (the step which uploads the rest of the steps)

Reply:
maybe add a comment that we want to run the purge script on windows to make it a bit less confusing